src/sections/Learn-Layer5/index.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { LearnLayer5GridWrapper , CardComponentWrapper} from "./learnlayer5.style";
 import { graphql, useStaticQuery } from "gatsby";
 import CardComponent from "../../components/Learn-Components/Card-Component";
+import Info from "./Info/index";
```

Comment:
Let's rename the component to `LearnInfo`

Reply:
Yep, that sounds a reasonable name :)